File: Said_Coley_Romero/Programas/Dart/Trabajo_31_de_Mayo.dart
import 'dart:io';

void main() {
  print("Numero de estudiantes :");
  int numEstudiantes = 
 int.parse(stdin.readLineSync()!);

  for (int i = 1; i <= numEstudiantes; i++) {
    print("Nombre del estudiante :");
    String nombre = stdin.readLineSync()!;

    print("Nota 1:");
    double nota1 = double.parse(stdin.readLineSync()!);
    print("Nota 2:");
    double nota2 = double.parse(stdin.readLineSync()!);
    print("Nota 3:");
    double nota3 = double.parse(stdin.readLineSync()!);

    double promedio = (nota1 + nota2 + nota3) / 3;
    if (promedio >= 5) {
    print("Excelente, aprobado con $promedio");
    } else {
    print("Su nota es baja, no aprobó. Promedio: $promedio");
    }
  }
}

File: Said_Coley_Romero/Programas/Ruby/Trabajo_31_de_Mayo.rb
puts "Número de estudiantes:"
num_estudiantes = gets.to_i

for i in 1..num_estudiantes do
    puts "Nombre del estudiante:"
    nombre = gets.chomp
    
    puts "Nota 1:"
    nota1 = gets.to_f
    puts "Nota 2:"
    nota2 = gets.to_f
    puts "Nota 3:"
    nota3 = gets.to_f

    promedio = (nota1 + nota2 + nota3) / 3.0

    if promedio >= 5
        puts "Excelente; aprobado con #{promedio}"
    else
        puts "Su nota es baja, no aprobó. Promedio: #{promedio}"
    end
end